source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -213,7 +213,6 @@ public void create() {
   private void displayUI() {
     Entity ui = new Entity();
     ui.addComponent(new GameAreaDisplay("Box Forest"));
-    ui.addComponent(ServiceLocator.getGameEndService().getDisplay());
```

Comment:
You have deleted a change required by another team's feature. Restore this and fix your error properly